torch/autograd/functional.py

```diff
@@ -274,7 +275,7 @@ def vjp(func, inputs, v=None, create_graph=False, strict=False):
     return _tuple_postprocess(outputs, is_outputs_tuple), _tuple_postprocess(vjp, is_inputs_tuple)
 
 
-def jvp(func, inputs, v=None, create_graph=False, strict=False):
+def jvp(func, inputs, v=None, create_graph=False, strict=False, fw_mode=False):
```

Comment:
The flag is called `forward_mode` in gradcheck -- we should probably be consistent? Also "fw" might not be the best abbreviation, if we wanted to abbreviate it "fwd" should be clearer